common/lib/xmodule/xmodule/js/spec/video/initialize_spec.js

```diff
@@ -222,8 +222,7 @@ function(Initialize) {
                 it('check mapping', function() {
                     var map = {
                         0.25: '0.75',
-                        '0.50': '0.75',
-                        '2.0': '1.50'
```

Comment:
Same question as below - why was this line removed?

Reply:
Actually interesting - when I add this back in it fails because HTML5 now can go to 2.0x, it doesn't need to fall back to 1.5x. So I'm removing this test.